src/utils/crypto.ts

```
@@ -1,35 +1,46 @@
+
+  static encodeKey(key: string, type: 'pub' | 'sec'): string {
+    return type === 'pub' ? nip19.npubEncode(key) : nip19.nsecEncode(hexToBytes(key));
+  }
+
+  static deriveConversationKey(privateKey: string, publicKey: string): Uint8Array {
+    const privKeyBytes = hexToBytes(privateKey);
+    return nip44.v2.utils.getConversationKey(privKeyBytes, publicKey);
+  }
+
+  static encryptMessage(plaintext: string, privateKey: string, publicKey: string): string {
+    const conversationKey = this.deriveConversationKey(privateKey, publicKey);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)